region/norfair/west.json

```diff
@@ -223,6 +132,681 @@
+                  "notable": false,
+                  "requires": [],
+                  "exitCondition": {
+                    "leaveWithRunway": {
+                      "length": 9,
+                      "openEnd": 0,
+                      "gentleDownTiles": 4
+                    }
+                  }
+                }
+              ]
+            },
+            {
+              "id": 5,
+              "strats": [
+                {
+                  "name": "Leave Shinecharged",
+                  "notable": false,
+                  "entranceCondition": {
+                    "comeInShinecharging": {
+                      "length": 8,
+                      "openEnd": 0,
+                      "gentleUpTiles": 4
+                    }
+                  },
+                  "requires": ["canShinechargeMovementComplex"],
+                  "exitCondition": {
+                    "leaveShinecharged": {
+                      "framesRemaining": 20
+                    }
+                  }
+                },
+                {
+                  "name": "Leave Shinecharged with HiJump",
+                  "notable": false,
+                  "entranceCondition": {
+                    "comeInShinecharging": {
+                      "length": 8,
+                      "openEnd": 0,
+                      "gentleUpTiles": 4
+                    }
+                  },
+                  "requires": [
+                    "canShinechargeMovementComplex",
+                    "HiJump"
+                  ],
+                  "exitCondition": {
+                    "leaveShinecharged": {
+                      "framesRemaining": 35
+                    }
+                  }
+                }
+              ]
+            },
+            {
+              "id": 8,
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": []
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "from": 5,
+          "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                  "name": "Leave Sparking",
+                  "notable": false,
+                  "entranceCondition": {
+                    "comeInShinecharging": {"length": 5,"openEnd": 1}
+                  },
+                  "requires": [
+                    "HiJump",
+                    "canMidairShinespark",
+                    "canShinechargeMovementComplex",
+                    "canFastWalljumpClimb",
+                    {"shinespark": {"frames": 10}}
+                  ],
+                  "exitCondition": {"leaveWithSpark": {}},
+                  "note": "Quickly climb the right side of the room to just have enough time to shinespark out the door.",
+                  "devNote": "Fast Climbing is required but you can make it up without walljumping."
```

Comment:
is there another tech we could put with an `or` to represent this? so it can be used on wall jump-less seeds?

Reply:
`canLedgeGrab` could be a new `Expert` level tech to add to indicate fast platforming without walljumps.  Logically, a ledge grab could only ever mean effecient movement, probably.  But I'm also interested in adding a `canInsaneJump`  which could end up being more general.  So I don't quite want to add a new tech with this PR.

Reply:
I think this would be a good idea. Maddo was wondering about a different name for it and for `canInsaneWalljump`